java/src/jmri/jmrix/loconet/messageinterp/MessageInterpretation_nl.properties

```diff
@@ -487,6 +488,48 @@ LN_MSG_OPC_MULTI_SENSE_TRANSP_ZONEH = H
 # replaced by a decimal representation of the LS 4 bits of byte 2 of the message.
 LN_MSG_OPC_MULTI_SENSE_TRANSP_ZONE_UNKNOWN = (Unknown {0})
 
+# Note: in # LN_MSG_OPC_MULTI_TRANSP_RAILCOM_REPORT, the following parameter
+# substitutions will be made:
+#   {0} is a string representing the decoder address which triggered the report,
+#       and whether that address is 'long' or 'short', from
+#       LN_MSG_LOCO_ADDRESS_HELPER_MIXED_ADDRESS or
+#       LN_MSG_LOCO_ADDRESS_HELPER_UNMIXED_ADDRESS, as appropriate
+#   {1} is a number representing the RailCom App:Dyn Index value
+#   {2} is a number representing the RailCom Dynamic Value
+LN_MSG_OPC_MULTI_SENSE_TRANSP_RAILCOM_REPORT = Transponder address {0} reporting {1}.\n
+
+# Note: in # LN_MSG_RAILCOM_REPORT, the following parameter
+# substitutions will be made:
+#   {0} is a number representing the RailCom App:Dyn Index value
+#   {2} is a string representing the RailCom App:Dyn Index value
+#   {3} is a number representing the RailCom Dynamic Value
```

Comment:
Same comment as for .properties

Reply:
Added usage comments and included the words _HELPER for this and LN_MSG_DIRECTION_WEST